docs/android/inpath/propertyDescriptions.md

```diff
@@ -119,6 +119,41 @@ A string value that represents the Order ID for a Flight PNR or Booking Referenc
 ### passenger
 An optional CTSdkPassenger, passing this will pre-populate the driver details form during the booking flow.
 
+The CTSdkPassenger object has the following optional parameters:
+
+| Parameter          | Description                                  | Usage                                                                                                                                       | Type      | 
+|:-------------------|:---------------------------------------------|:--------------------------------------------------------------------------------------------------------------------------------------------|-----------|
+| firstName          | Customer's name                              | First name displayed on the driver details' screen                                                                                          | String    |
+| lastName           | Customer's last name                         | Last name displayed on the driver details' screen                                                                                           | String    |
+| email              | Customer's email                             | Email displayed on the driver details' screen                                                                                               | String    |
```

Comment:
I don't think driver details' should have the apostrophe at the end in any of these entries

Reply:
done